pythonforandroid/bootstraps/sdl2/build/jni/Application.mk

```diff
@@ -5,3 +5,4 @@
 
 # APP_ABI := armeabi armeabi-v7a x86
 APP_ABI := $(ARCH)
+APP_PLATFORM := android-19
```

Comment:
Setting this to a fixed value for everyone could possibly have some surprising consequences, and there is the problem that it probably also acts as a minimum supported api. It might be better to leave this change until a more robust implementation is complete. Alternatively, if targeting a very old api works (like 12) that could be okay, I'd still be wary of surprising consequences.

Reply:
I just taken this from your branch. Not yet in state where I tried to upload to playstore. Once it works with this changes, I'll try to upload an apk without this change.

Reply:
I think you will need to do either this, or the create-dist-with-api-19-then-delete-and-rebuild-with-api-27 method suggested in the other thread. However, the problem is that hardcoding it to a single value will probably cause some problems itself - it didn't matter in my branch or for individual testing, but it's the kind of thing that causes issues when dispersed across the wide range of configurations people use.

Reply:
It would be great to get a working apk without hacks like delete-and-rebuild. Let's see what playstore tells me for an apk build like it's done now, and think about generalization afterwards

Reply:
this one still needs to be tested